scripts/binary/trigger-publish-binary-pipeline.js

```diff
@@ -10,6 +10,7 @@ const { getNextVersionForBinary } = require('../get-next-version')
   const { nextVersion } = await getNextVersionForBinary()
 
   const body = JSON.stringify({
+    branch: 'ryanm/fix/better-sqlite3',
```

Comment:
Does this need to be removed before this is merged?

Reply:
Yep. Good reminder.

Reply:
```suggestion
```